processor/src/main/java/com/blarley/rxretrobusseed/annotationprocessor/processor/RxRetroBusAnnotationProcessor.java

```diff
@@ -27,20 +27,22 @@
     @Override
     public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment roundEnv) {
 
-        List<String> generatedClasses = new ArrayList<>();
+        Map<String, Boolean> generatedClasses = new ConcurrentHashMap<>();
```

Comment:
Don't misuse a map like this. 
Create a class that holds a GenerateEvents object and a string for the classname and leave it as a List<ThatClass>